README.md

```diff
@@ -2,29 +2,29 @@
 
 # Databraid Sheet API
 
-A part of the *Databraid Dashboard*
 ## Description
-This is an API for the Google Sheets widget. It represents a microservice that functions indepently or as part of the [Databraid Dashboard](https://github.com/databraid-dashboard/dashboard-spa). The Sheets widget leverages [GraphQL](http://graphql.org/) to create a powerful and fast singular endpoint to access the [Google Sheets API](https://developers.google.com/sheets/api/). With this widget, you can project your most relevant Google Spreadsheet file into your dashboard and keep up with changes in real time.  
+This is an API for the Google Sheets widget. It represents a microservice that functions in conjunction with the [Sheets Single Page Application](https://github.com/databraid-dashboard/sheet-spa) or as part of the [Databraid Dashboard](https://github.com/databraid-dashboard/dashboard-spa). This API leverages [GraphQL](http://graphql.org/) to create a powerful and fast singular endpoint to access the [Google Sheets API](https://developers.google.com/sheets/api/). With this widget, you can project your most relevant Google Spreadsheet file into your dashboard and keep up with changes in real time.  
```

Comment:
"powerful and fast" 🐬

Reply:
@Gwillison415 unfortunately we have not yet had the chance to "keep up with changes in real time."